### DISCUSSION TITLE
Custom layout, more than one component on one page?

### DISCUSSION THREAD

**Discussion by GBeushausen** [filamentphp/filament#11462] [Help]
### Package

Panel builder

### Package Version

v3.2.2

### How can we help you?

Hi,
i've played a bit with Filament 3 and so far it's totally awesome. But i wonder if it's possible to customize the layout or add more than one component in one page. For example take a CRM, where one can view a company, but on the left side there is a bar where you can also see all contacts / employees of the company and in the main screen, you got the company data, but below you can scroll down and also see all activities, emais, orders etc all in one place. Is this possible with Filament and if so can you guide me in a direction where i can see how? Thanks.

**Comment by leandrocfe** [comment]
I believe that you can use [layout components](https://filamentphp.com/docs/3.x/forms/layout/getting-started) and [relation managers](https://filamentphp.com/docs/3.x/panels/resources/relation-managers) to achieve what you need
